runtime/src/storage/mkvs/urkel/tree/tree_test.rs

```diff
@@ -716,6 +716,37 @@ fn test_syncer_insert() {
     assert_eq!(0, stats.sync_iterate_count, "sync_iterate count");
 }
 
+#[test]
+fn test_syncer_writelog_remove() {
+    let server = ProtocolServer::new();
+
+    let mut tree = UrkelTree::make()
+        .with_capacity(0, 0)
+        .new(Box::new(NoopReadSyncer {}));
+
+    let (keys, values) = generate_key_value_pairs();
+    for i in 0..keys.len() {
+        tree.insert(
+            Context::background(),
+            keys[i].as_slice(),
+            values[i].as_slice(),
+        )
+        .expect("insert");
+    }
+
+    let (write_log, hash) =
+        UrkelTree::commit(&mut tree, Context::background(), Default::default(), 0).expect("commit");
+    server.apply(&write_log, hash, Default::default(), 0);
+
+    tree.remove(Context::background(), keys[0].as_slice())
+        .expect("remove");
+
+    let previous_hash = hash;
+    let (write_log, hash) =
+        UrkelTree::commit(&mut tree, Context::background(), Default::default(), 0).expect("commit");
+    server.apply_existing(&write_log, previous_hash, hash, Default::default(), 0);
```

Comment:
:information_source: smoketest only

Reply:
Yes this tests that write logs are compatible by sending them to the protocol server and making sure that the computed hash matches. This caught the write log format change.

I'll add a comment.